index.js

```diff
@@ -56,6 +56,25 @@ module.exports = {
     'import/no-named-default': 'error',
     'import/no-webpack-loader-syntax': 'error',
     'import/prefer-default-export': 'error',
+    'jsdoc/check-examples': 0,
```

Comment:
I would turn it on. It confirms that eventual examples are validated by lint, which seems to make sense

Reply:
I can change this. This tag hasn't been used much.

Reply:
fixed